app/models.py

```diff
@@ -1,40 +1,55 @@
 from app import db
 from datetime import datetime
 
+
 class User(db.Model):
     __tablename__ = "user"
 
-    id = db.Column(db.Integer, primary_key = True)
+    id = db.Column(db.Integer, primary_key=True)
     username = db.Column(db.String(40), nullable=False)
     email = db.Column(db.String(40), nullable=False)
     password = db.Column(db.String(256), nullable=False)
-    date_created = db.Column(db.DateTime, nullable=False, default = datetime.utcnow)
+    date_created = db.Column(
+        db.DateTime, nullable=False, default=datetime.utcnow
+    )
+    date_modified = db.Column(db.DateTime, onupdate=datetime.utcnow)
     budgets = db.relationship('Budget', backref='user', lazy=True)
+    # QUESTION: Why do you prefer lazy=True?
```

Comment:
To add to this. I found a video that helps explain SQLAlchemy loading techniques...
[here](https://www.youtube.com/watch?v=g1oFlq7D_nQ)